src/test/java/SearchServiceFacetsRestrictedValuesTest.java

```diff
@@ -14,66 +14,66 @@
 import java.util.concurrent.TimeUnit;
 
 public class SearchServiceFacetsRestrictedValuesTest {
-    //TODO(developers) change the values of following variables to use proper project
-    private static final long YOUR_PROJECT_NUMBER = 00000L;
+  //TODO(developers) change the values of following variables to use proper project
+  private static final long YOUR_PROJECT_NUMBER = 00000L;
 
-    private static final String ENDPOINT = "retail.googleapis.com:443";
-    private static final String DEFAULT_CATALOG_NAME =
-            String.format("projects/%d/locations/global/catalogs/default_catalog", YOUR_PROJECT_NUMBER);
-    private static final String DEFAULT_SEARCH_PLACEMENT_NAME =
-            DEFAULT_CATALOG_NAME + "/placements/default_search";
+  private static final String ENDPOINT = "retail.googleapis.com:443";
+  private static final String DEFAULT_CATALOG_NAME =
+      String.format("projects/%d/locations/global/catalogs/default_catalog", YOUR_PROJECT_NUMBER);
+  private static final String DEFAULT_SEARCH_PLACEMENT_NAME =
+      DEFAULT_CATALOG_NAME + "/placements/default_search";
 
-    private static final String VISITOR_ID = UUID.randomUUID().toString();
+  private static final String VISITOR_ID = UUID.randomUUID().toString();
 
-    // [START search_client]
-    private static SearchServiceClient getSearchServiceClient() throws IOException {
-        SearchServiceSettings settings = SearchServiceSettings.newBuilder()
-                .setEndpoint(ENDPOINT)
-                .build();
-        return SearchServiceClient.create(settings);
-    }
-    // [END search_client]
+  // [START search_client]
```

Comment:
please remove excessive region tags [START/END...] and add them before import and in the end of code sample